rollup.config.js

```diff
@@ -14,21 +14,8 @@ const compiledLicense = _.template(fs.readFileSync('./build/license-header.txt',
 const bannerData = _.pick(pkg, ['version', 'copyright']);
 const banner = compiledLicense(Object.assign({includesVtt: true}, bannerData));
 
-const watch = {
-  clearScreen: false
```

Comment:
I don't think we want to go into a screen ever, so I added the fix from `videojs-generate-rollup-config`

Reply:
Why isTTY over clearScreen? I would think it would be better to use the rollup option over modifying the `isTTY` option, especially since clearScreen should be working (I fixed it).

Reply:
There isn't an option to disable screen usage when doing rollup outside of watch. The last time I checked it was still going into a screen for builds.

Reply:
Does it matter if we go into screen mode outside of watch? Also, I don't think it actually uses the alternate screen during regular builds, it should only be happening during watches.

Reply:
I guess it doesn't matter too much, it just messed with scroll back history

Reply:
I will have to test and see if it still uses a screen during build.

Reply:
Just tried it out and rollup doesn't use the second screen during regular builds, it only uses it during watch builds. This is disabled with `clearScreen: false`.